packages/fx-core/src/plugins/solution/fx-solution/v2/executeUserTask.ts

```diff
@@ -1038,18 +1052,64 @@ export async function createAuthFiles(
   }
 
   if (needBot) {
-    const botFolder = path.join(authFolder, "bot");
+    const botFolder = path.join(authFolder, AddSsoParameters.Bot);
     const botFolderExists = await fs.pathExists(botFolder);
     if (!botFolderExists) {
       await fs.ensureDir(botFolder);
     }
 
-    // TODO: Add necessary files here for bot
+    try {
+      const templateFolder = getTemplatesFolder();
+      const botTemplateFolder = path.join(
+        templateFolder,
+        AddSsoParameters.filePath,
+        AddSsoParameters.Bot
+      );
+      if (isVsProject) {
+        // TODO: add steps for VS
+      } else {
+        // README.md
+        const readmeSourcePath = path.join(botTemplateFolder, AddSsoParameters.Readme);
+        const readmeTargetPath = path.join(botFolder, AddSsoParameters.Readme);
+        const readme = await fs.readFile(readmeSourcePath);
+        fs.writeFile(readmeTargetPath, readme);
+
+        // Sample Code
+        const sampleSourceFolder = path.join(botTemplateFolder, languageFolderName);
+        const sampleZip = new AdmZip();
+        sampleZip.addLocalFolder(sampleSourceFolder);
+        await unzip(sampleZip, botFolder);
+      }
+    } catch (error) {
+      const e = new SystemError(
```

Comment:
A suggestion: if we failed to add resources, we'd better to remove all the file we added